src/main/java/org/opensearch/flowframework/transport/GetWorkflowStateRequest.java

```diff
@@ -17,9 +17,9 @@
 import java.io.IOException;
 
 /**
- * Transport Request to get a workflow or workflow status
+ * Transport Request to get a workflow status
  */
-public class GetWorkflowRequest extends ActionRequest {
+public class GetWorkflowStateRequest extends ActionRequest {
```

Comment:
We now have `workflowRequest` and `GetWorkflowStateRequest` , do you think we should have a separate `GetWorkflowRequest` to differentiate get and post/put (create)?

Reply:
Agreed, ideally we should separate out the `WorkflowRequest` class into `CreateWorkflowRequest` and `ProvisionWorkflowRequest` and have them used specifically for their corresponding transport actions. But that is out of the scope of this PR. which is only for the Get Workflow API

Reply:
we also should have a getWorkRequest right eventually?